subsys/bluetooth/audio/vocs.c

```diff
@@ -53,7 +83,8 @@ static ssize_t write_location(struct bt_conn *conn, const struct bt_gatt_attr *a
 			      const void *buf, uint16_t len, uint16_t offset, uint8_t flags)
 {
 	struct bt_vocs *inst = BT_AUDIO_CHRC_USER_DATA(attr);
-	uint32_t old_location = inst->srv.location;
+	enum bt_audio_location new_location;
+	uint32_t rfu_locations = ~((uint32_t)BT_AUDIO_LOCATION_ALL);
```

Comment:
If the RFU locations is really what we want, we should have that as a define as well (or instead).

Reply:
@Thalley nah, everything that is not defined in `enum bt_audio_location` can be considered as RFU, thus I'd not define RFU values explicitly

Reply:
I'm not suggesting that we in `audio.h` defined the RFU values, but IMO it makes more sense to use a `#define` for the RFU values instead of a local variable.

Reply:
@Thalley I understand now, makes sense indeed :)

Reply:
@MariuszSkamra @Thalley I'm not sure if I'm defining this macro now correctly. Is this the proper way?